packages/react-positioning/src/createArrowStyles.ts

```diff
@@ -0,0 +1,80 @@
+import type { MakeStylesStyleRule } from '@fluentui/make-styles';
+import type { Theme } from '@fluentui/react-theme';
+
+/**
+ * Helper that creates a makeStyles rule for an arrow element.
+ * For runtime arrow size toggling simply create extra classnames to apply to the arrow element
+ *
+ * ```ts
+ *   makeStyles({
+ *     arrowWithSize: createArrowStyles(5),
+ *
+ *     arrowWithoutSize: createArrowStyles(5),
```

Comment:
I'm a little confused by this... is it meant to have no size?
```suggestion
 *     arrowWithoutSize: createArrowStyles(),
```

Reply:
@behowell  exactly. The size in this helper can only be applied at build time. In the case of `Tooltip` this is fine. But for `Popover` there are sizes which can be toggled at runtime which means that it needs to be applied instead with different classnames. 

I could also just set the default here too for clarity

Reply:
I understood about providing size as an argument :), but the example in the comment was confusing. Is it a typo that it included a size of 5 in the `arrowWithoutSize` example?

Reply:
oh whoops, yeah that's a good catch